react-frontend/src/shared-components/peerly-logo/LogoComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { Col, Row } from "react-bootstrap";
+import Styled from "styled-components";
+
+import LabelComponent from "core-components/Label/LabelComponent";
+
+const Logo = Styled.div`
```

Comment:
React developer guidelines not met. Go through them again.

Reply:
@onkar-josh address this comment

Reply:
@mayuriardad the folder name of label is changed.